ignite/services/network/share_percent_test.go

```diff
@@ -0,0 +1,179 @@
+package network
```

Comment:
```suggestion
package network_test
```

and refactor tests so we don't depend on private values